pkg/da/data_availability_header_test.go

```diff
@@ -25,14 +25,11 @@ func TestNilDataAvailabilityHeaderHashDoesntCrash(t *testing.T) {
 func TestMinDataAvailabilityHeader(t *testing.T) {
 	dah := MinDataAvailabilityHeader()
 	expectedHash := []byte{
-		0x6f, 0x52, 0xda, 0xc1, 0x65, 0x45, 0xe4, 0x57, 0x25, 0xbe, 0x6e, 0xa3, 0x2a, 0xed, 0x55, 0x26,
-		0x6e, 0x45, 0x3, 0x48, 0x0, 0xee, 0xe1, 0xd8, 0x7c, 0x94, 0x28, 0xf4, 0x84, 0x4e, 0xa4, 0x7a,
+		0x25, 0x77, 0x60, 0x46, 0x19, 0x93, 0xf8, 0xf1, 0x97, 0xb4, 0x21, 0xec, 0x74, 0x35, 0xf3, 0xc3,
+		0x6c, 0x37, 0x34, 0x92, 0x3e, 0x3d, 0xa9, 0xa4, 0x2d, 0xc7, 0x3b, 0x5, 0xf0, 0x7b, 0x3d, 0x8,
 	}
 	require.Equal(t, expectedHash, dah.hash)
 	require.NoError(t, dah.ValidateBasic())
-	// important note: also see the types.TestEmptyBlockDataAvailabilityHeader test
-	// which ensures that empty block data results in the minimum data availability
-	// header
```

Comment:
[for reviewers] I deleted this comment because I could not find the test that it referred to (in celestia-app or celestia-core)

Reply:
yeah, it was a remnant of when this code was in core